Dockerfile

```diff
@@ -1,12 +1,17 @@
 FROM python:3.8-slim-buster
-MAINTAINER Marcin Sztolcman <marcin@urzenia.net>
+LABEL maintainer="Marcin Sztolcman <marcin@urzenia.net>"
 
+RUN python3 -m pip install pipenv
 RUN useradd --create-home sendria
-WORKDIR /home/sendria
 USER sendria
-RUN python3 -m pip install --user sendria==2.1.0
+COPY --chown=sendria:sendria . /home/sendria/sendria
+WORKDIR /home/sendria/sendria
+RUN pipenv lock --requirements > requirements.txt
+RUN python3 -m pip install --user -r requirements.txt
```

Comment:
Why install in this way? Why not just `pipenv install`?

Reply:
I'm not a Pipenv expert (I mainly use poetry), but I guess that's possible.
That would imply that the CMD would be along the lines of "pipenv run sendria".
I had the idea of adding sendria to the PATH, so there is the need to change the arguments, you wouldn't have to remember tedious commands.
Using pipenv would throw a wrench in that plan I believe.